wolfcrypt/src/curve25519.c

```diff
@@ -51,12 +51,45 @@ const curve25519_set_type curve25519_sets[] = {
     }
 };
 
+/* compute the public key from an existing private key, using bare vectors. */
+int wc_curve25519(int public_size, byte* public, int private_size, const byte* private) {
+    int ret;
+
+    if ((public_size != CURVE25519_KEYSIZE) ||
+        (private_size != CURVE25519_KEYSIZE))
```

Comment:
I recommend adding braces even for the one line since the expression is more than one line.

Reply:
done